.github/workflows/overall-tests.yml

```diff
@@ -26,88 +27,48 @@ jobs:
         with:
           python-version: ${{matrix.python-version}}
 
-      - name: Install pipenv
-        run: curl https://raw.githubusercontent.com/pypa/pipenv/master/get-pipenv.py | python
-
-      - name: Install Dependencies
-        run: pipenv install --dev --python=${{ matrix.python-version }}
-
-      - name: Setup LibMagic (MacOS)
-        if: matrix.os == 'macos-latest'
-        run: brew install libmagic
-
-      - uses: actions/setup-node@v4
+      - name: Install dependencies
+        id: install-dependencies
+        uses: ./.github/actions/install
         with:
-          node-version: 20
-      - name: Frontend Bundle Build
-        run: pipenv run python tools/frontend/bundle_build.py
-
-      - name: Install Playwright
-        run: pipenv run playwright install chromium --with-deps
+          python-version: ${{ matrix.python-version }}
+          os: ${{ matrix.os }}
+          pipfile-version: ${{ matrix.pipfile-version }}
 
       - name: Pytest
         run: pipenv run pytest -m "not orchestrator_dispatcher and not standalone" --cov=taipy --cov-append --cov-report="xml:overall-coverage.xml" --cov-report term-missing tests
 
       - name: Coverage
-        if: matrix.os == 'ubuntu-latest' && matrix.python-version == '3.11'
+        if: matrix.os == 'ubuntu-latest' && matrix.python-version == '3.11' && matrix.pipfile-version == 'min'
         uses: MishaKav/pytest-coverage-comment@main
         with:
           pytest-xml-coverage-path: ./overall-coverage.xml
           title: Taipy Overall Coverage Report
 
-  submit_tests:
+  orchestrator-tests:
```

Comment:
We can rename this job `intermittent-tests`

Reply:
Because they are separated from the others because they are intermittent. They fail time to time in an unpredictable way.